test/markup/shell/prompt-with-slash.txt

```diff
@@ -1,6 +1,6 @@
-pao-sw-a04-40(config)# interface range ethernet 1/1/1-1/1/9
-pao-sw-a04-42(conf-range-eth1/1/1-1/1/9)# switchport trunk allowed vlan 1102
-pao-sw-a04-42(conf-range-eth1/1/1-1/1/9)# show configuration
+pao.sw.a04-40(config)# interface range ethernet 1/1/1-1/1/9
+pao.sw.a04-42(conf-range-eth1/1/1-1/1/9)# switchport trunk allowed vlan 1102
+pao.sw.a04-42(conf-range-eth1/1/1-1/1/9)# show configuration
```

Comment:
Why would we remove the old tests?  The goal here is to support both, yes?  If so lets put the old tests back so that we're testing both formats.  

Otherwise seems reasonable. Thanks!

Reply:
Since this replaces a “-“ with a “.” but retains other “-“ it feels to me this covers both cases OK.  If you feel strongly I can make another separate test.

Reply:
Yes, lets keep the old tests and just add the 3 new lines of tests right after.  That'd be perfect!

Reply:
Going back and re-opening the file I realised it's called `prompt-with-slash.txt` and there's other tests named like that -- so I agree that overloading it to test multiple things isn't keeping with what's there.  So switched to a individual test as first suggested.